raiden/raiden_service.py

```diff
@@ -647,14 +1070,17 @@ def __init__(self, raiden):
 
     def on_message(self, message, msghash):  # noqa pylint: disable=unused-argument
         """ Handles `message` and sends an ACK on success. """
+        if log.isEnabledFor(logging.INFO):
+            log.info('message received', message=message)
+
         cmdid = message.cmdid
 
         # using explicity dispatch to make the code grepable
         if cmdid == messages.ACK:
             pass
 
         elif cmdid == messages.PING:
-            self.message_ping(message)
+            pass
```

Comment:
All methods in this handler are named `message_<message_type>`, the `message_ping` method was a empty function definition with a single `pass` statement, this just removed the unnecessary method call.

Pings are answered with an ACK by the protocol class